ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -105,6 +105,26 @@ EOF
 
 echo "CatalogSource name is \"$CATSRC\""
 
+IS_CATSRC_CREATED=false
+# Wait for 10 minutes until the Catalog source state is 'READY'
+for i in $(seq 1 120); do
+    CATSRC_STATE=$(oc get catalogsources/"$CATSRC" -n "$OO_INSTALL_NAMESPACE" -o jsonpath='{.status.connectionState.lastObservedState}')
+    echo $CATSRC_STATE
+    if [ "$CATSRC_STATE" = "READY" ] ; then
+	echo "CATALOG SOURCE CREATED SUCCESSFULLY AFTER WAITING FOR $((5*i)) SECONDS"
+        echo "CURRENT STATE OF CATALOG SOURCE is \"$CATSRC\""
+        IS_CATSRC_CREATED=true
+        break
+    fi
+    sleep 5
+done
+
+if [ $IS_CATSRC_CREATED = false ] ; then
+    echo "CATALOG SOURCE CREATION FAILED AFTER WAITING FOR 10 MINUTES"
```

Comment:
Since we can't know if the creation failed outright, can you change this message to something like:
```
Timed out waiting for the catalog source "${CATSRC}" to become ready after 10 minutes.
```